draft-ietf-opsawg-pcapng.md

```diff
@@ -2223,6 +2223,48 @@ The following is a list of Secrets Types.
 {: #format_zigbee_aps title='ZigBee APS Key Data Format' artwork-align="center"}
 
 
+{: indent='8'}
+0x45535053:
+: ESP SAs UAT.
+  This format uses a CSV file with strings describing an ESP security association
```

Comment:
I'd say "uses CSV format ([RFC4180])", as it's not a film by itself, it's a block in a pcapng file; that also adds a reference to RFC 4180 "Common Format and MIME Type for Comma-Separated Values (CSV) Files" (yes, it says "Files", but the "Common Format" can also be used for embedded CSV text).

Reply:
Maybe "this is in CSV format ([RFC 4180]), with each record containing fields that describe an ESP security association".

[RFC 4180](https://datatracker.ietf.org/doc/html/rfc4180) says that the header line, with column names, is optional; is it optional or required here?